core-bundle/contao/dca/tl_content.php

```diff
@@ -602,6 +602,12 @@
 			'eval'                    => array('includeBlankOption' => true, 'nospace'=>true, 'tl_class'=>'w50'),
 			'sql'                     => "varchar(8) COLLATE ascii_bin NOT NULL default ''"
 		),
+		'textTrackSRC' => array
+		(
+			'inputType'               => 'fileTree',
+			'eval'                    => array('multiple'=>true, 'fieldType'=>'checkbox', 'submitOnChange'=>true, 'filesOnly'=>true, 'extensions'=>'vtt', 'isSortable'=>true),
```

Comment:
```suggestion
			'eval'                    => array('multiple'=>true, 'fieldType'=>'checkbox', 'filesOnly'=>true, 'extensions'=>'vtt', 'isSortable'=>true),
```

This does not seem to be necessary and is otherwise irritating, as `submitOnChange` on non-checkboxes cause a complete reresh.